.bazelrc

```diff
@@ -1,5 +1,5 @@
 # Envoy specific Bazel build/test options.
-
+startup --host_jvm_args=-Xmx2048m
```

Comment:
How does this help if Bazel allocates memory above this? Won't it still OOM? Or does this force GC more often as it approaches the limit? Can you add a comment on how this was selected?

Reply:
No it won't, this is to control GC, added comment.

Reply:
If it doesn't need more than 200MB, why are we giving it 2GB? To account for GC slop?

Reply:
I don't have specific reason for that, let's try lower number :)